### PR TITLE
frontend/#2470/scroll view focus on selected child

### PR DESCRIPTION
**Pull request by ashishsingh101**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs [hunk 1]
```diff
@@ -1804,14 +1808,20 @@ eval GoToEditProfile state = do
   exit $ GoToMyProfile state true
 eval (MenuButtonActionController (MenuButtonController.OnClick config)) state = do
   continueWithCmd state{props{defaultPickUpPoint = config.id}} [do
+      let focusedIndex = findIndex (\item -> item.place == config.id) state.data.nearByPickUpPoints
```
Comment (Vignesh-772) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs:1811]: Is it required in `onClick` also?

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs [hunk 2]
```diff
@@ -2361,17 +2361,24 @@ currentLocationView push state =
 
 nearByPickUpPointsView :: forall w . HomeScreenState -> (Action -> Effect Unit) -> PrestoDOM (Effect Unit) w
 nearByPickUpPointsView state push =
-  linearLayout
-    [ height WRAP_CONTENT
+  scrollView
+    [ height (V 130)
```
Comment (Vignesh-772) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs:2365]: don't hard code the height try to use `getLayoutBounds` to update layout bounds dynamically.

**Review by Vignesh-772 — CHANGES_REQUESTED** [review]
Check the comments

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs (hunk 1 above)
Comment (ashishsingh101) [reply Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs:1811]: yes

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs (hunk 2 above)
Comment (ashishsingh101) [reply Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs]: done

**Review by Vignesh-772 — APPROVED** [review]

**Pull request closed without merging**